24-inf-model-slr.qmd

```diff
@@ -495,7 +519,7 @@ lm(weight ~ mage, data = births14_100) %>%
 
 ### Variability of the statistic
 
-Because the focus here is *not* on a null distribution, sample with replacement $n=100$ observations from the original dataset.
+Because the focus here is *not* on a null distribution, we sample with replacement $n=100$ observations from the original dataset.
```

Comment:
```suggestion
Because the focus here is *not* on a null distribution, we sample with replacement $n = 100$ observations from the original dataset.
```